composer.json

```diff
@@ -35,12 +35,12 @@
 	},
 	"require": {
 		"php": ">=7.0.0",
```

Comment:
With the upgrade to CakePHP 4.0, the minimum PHP version becomes [PHP 7.2](https://packagist.org/packages/cakephp/cakephp#4.0.0) and starting with CakePHP 4.4.0 it becomes even [PHP 7.4](https://packagist.org/packages/cakephp/cakephp#4.4.0).

Depending on what CakePHP version we want to depend on, the correct PHP version should be set.

We should mention the PHP requirement bump in the change log, too.